paf-mvp-core-js/src/express/node.ts

```diff
@@ -236,6 +238,11 @@ export class Node implements INode {
       next();
     };
 
+  private logError(endpointName: string, error: NodeError, correlationId: string, sender: string) {
+    const errorMessage = `@${endpointName} --correlation-id=${correlationId} --type=${error.type} --details=${error.details} --sender=${sender}`;
```

Comment:
The pattern is weird here: in `beginHandling` and `endHandling`, you use `endpointName @correlationId`, here it is `@endpointName --correlation-id=correlationId`.
Can you propose something more consistent? At least, avoid having a `@` prefix for the endpoint name in one case, for the correlation id in the other case

Reply:
I'll keep the `@` in error logs because it makes sens (Error @...) and for consistency I will remove it from the begin and end handling